libipt/include/intel-pt.h.in

```diff
@@ -1451,7 +1451,7 @@ struct pt_event {
 
 		/** Event: tick. */
 		struct {
-			/** The instruction address near which the tick occured.
+			/** The instruction address near which the tick occurred.
```

Comment:
This moves the line beyond the 80 columns limit. Please change the formatting to stay within 80 columns.

Reply:
not sure I get it, I see only 60 symbols in this line...

Reply:
There are 8-space tabs in front.

Reply:
updated